pyDeltaRCM/bmi_delta.py

```diff
@@ -22,23 +23,119 @@ class BmiDelta(Bmi):
+            'type': 'bool', 'default': False},
+        'model_output__opt_stage_grids': {'name':'save_stage_grids',
+            'type': 'bool', 'default': False},
+        'model_output__opt_depth_grids': {'name':'save_depth_grids',
+            'type': 'bool', 'default': False},
+        'model_output__opt_discharge_grids': {'name':'save_discharge_grids',
+            'type': 'bool', 'default': False},
+        'model_output__opt_velocity_grids': {'name':'save_velocity_grids',
+            'type': 'bool', 'default': False},
+        'model_output__opt_time_interval': {'name':'save_dt',
+            'type': 'float', 'default': 50.0},
+        'coeff__surface_smoothing': {'name': 'Csmooth',
+            'type': 'float', 'default': 0.9},
+        'coeff__under_relaxation__water_surface': {'name': 'omega_sfc',
+            'type': 'float', 'default': 0.1},
+        'coeff__under_relaxation__water_flow': {'name': 'omega_flow',
+            'type': 'float', 'default': 0.9},
+        'coeff__iterations_smoothing_algorithm': {'name': 'Nsmooth',
+            'type': 'int', 'default': 5},
+        'coeff__depth_dependence__water': {'name': 'theta_water',
+            'type': 'float', 'default': 1.0},
+        'coeff__depth_dependence__sand': {'name': 'coeff_theta_sand',
+            'type': 'float', 'default': 2.0},
+        'coeff__depth_dependence__mud': {'name': 'coeff_theta_mud',
+            'type': 'float', 'default': 1.0},
+        'coeff__non_linear_exp_sed_flux_flow_velocity': {'name': 'beta',
+            'type': 'int', 'default': 3},
+        'coeff__sedimentation_lag': {'name': 'sed_lag',
+            'type': 'float', 'default': 1.0},
+        'coeff__velocity_deposition_mud': {'name': 'coeff_U_dep_mud',
+            'type': 'float', 'default': 0.3},
+        'coeff__velocity_erosion_mud': {'name': 'coeff_U_ero_mud',
+            'type': 'float', 'default': 1.5},
+        'coeff__velocity_erosion_sand': {'name': 'coeff_U_ero_sand',
+            'type': 'float', 'default': 1.05},
+        'coeff__topographic_diffusion': {'name': 'alpha',
+            'type': 'float', 'default': 0.1},
+        'basin__opt_subsidence': {'name':'toggle_subsidence',
+            'type': 'bool', 'default': False},
+        'basin__maximum_subsidence_rate': {'name': 'sigma_max',
+            'type': 'float', 'default': 0.000825},
+        'basin__subsidence_start_timestep': {'name': 'start_subsidence',
+            'type': 'float', 'default': 0},
+        'basin__opt_stratigraphy': {'name': 'save_strata',
+            'type': 'bool', 'default': False}
+        }
+
+
     def __init__(self):
         """Create a BmiDelta model that is ready for initialization."""
         self._delta = None
-        self._time = 0.
         self._values = {}
         self._var_units = {}
         self._grids = {}
         self._grid_type = {}
 
-    def initialize(self, filename='deltaRCM.yaml'):
+    def initialize(self, filename = 'deltaRCM.yaml'):
```

Comment:
spacing? not really important.

Reply:
It's just a preference of mine. Is the PEP8 requirement to not have that? I like it much better with space between binary operators including `=`

Reply:
It is pep8 to use spaces around binary operators everywhere except in parameter definitions. 
https://www.python.org/dev/peps/pep-0008/#other-recommendations

> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter: